RFC-008-ERC20.md

```diff
@@ -5,16 +5,12 @@
 - Discussion-issue: [#45](https://github.com/comit-network/RFCs/issues/45)
 - Created on: 2019-03-12
 
-## Table of Contents
-<!-- markdown-toc start -->
+**Table of Contents**
 - [Description](#description)
-- [The ERC20 Asset](#the-erc20-asset-1)
+- [The ERC20 Asset](#the-erc20-asset)
     - [`quantity`](#quantity)
-    - [`address`](#address)
+    - [`token_contract`](#tokencontract)
```

Comment:
this link does not work on github. I think it should be:
```suggestion
    - [`token_contract`](#token-contract)
```
or `#-token-contract-`

Reply:
Will fix this one!